examples/tfengine/generated/org_foundation/audit/main.tf

```diff
@@ -133,9 +151,20 @@ resource "google_project_iam_member" "logs_viewers_auditors" {
   member  = "group:${var.auditors_group}"
 }
 
-# IAM permissions to grant log Auditors iam.securityReviewer role to view the logs.
+# IAM organization permissions to grant log Auditors iam.securityReviewer role to view the logs.
```

Comment:
Organization level IAM permissions to ...

Reply:
Done